monitoring/registry.go

```diff
@@ -106,22 +106,52 @@ func (r *Registry) Get(name string) Var {
 
 // GetRegistry tries to find a sub-registry by name.
 func (r *Registry) GetRegistry(name string) *Registry {
-	e, err := r.find(name)
```

Comment:
The changes in GetRegistry aren't strictly needed (they're a functional no-op), just some cleanup to remove some duplicate lines while I'm in here